srcpkgs/deluge/template

```diff
@@ -1,20 +1,22 @@
 # Template file for 'deluge'
 pkgname=deluge
-version=2.0.3
-revision=15
-build_style=python3-module
+version=2.0.5
+revision=1
+build_style="python3-module"
 # TODO package python3-slimit to minify javascript
 hostmakedepends="intltool python3-setuptools python3-wheel"
 depends="python3-setuptools python3-chardet python3-Twisted python3-Mako
  python3-xdg python3-rencode python3-setproctitle libtorrent-rasterbar-python3
  python3-Pillow"
```

Comment:
```suggestion
depends="python3-setuptools python3-chardet python3-Twisted python3-Mako
 python3-xdg python3-rencode python3-setproctitle libtorrent-rasterbar-python3
 python3-Pillow python3-pyasn1 python3-openssl python3-six python3-zope.interface"
```
All of them are installed as dependencies of dependencies, but it's much better to have them listed explicitly when they are listed in setup.py.